.clang-format

```diff
@@ -78,6 +78,7 @@ IncludeCategories:
 IndentCaseLabels: false
 IndentWidth: 8
 # SpaceBeforeParens: ControlStatementsExceptControlMacros # clang-format >= 13.0
+InsertBraces: true
```

Comment:
IIRC we do not enable options for clang-format > 12 (see line above). This option was added in clang-format 15. We should probably ship latest version with sdk cc @stephanosio

Reply:
I thought we already settled on using the Python package.

See https://github.com/zephyrproject-rtos/sdk-ng/issues/500#issuecomment-1174870528

Reply:
> I thought we already settled on using the Python package.
> 
> See [zephyrproject-rtos/sdk-ng#500 (comment)](https://github.com/zephyrproject-rtos/sdk-ng/issues/500#issuecomment-1174870528)

Ah right. So then we should allow using latest options, right?

Reply:
> Ah right. So then we should allow using latest options, right?

Yes, I think that is reasonable.

Reply:
I understand correctly, enabling InsertBraces also requires a changes to the [requirements](https://github.com/zephyrproject-rtos/zephyr/blob/c0455b836fb6d56a9a68c0ef4e3b636d2837f35f/scripts/requirements-extras.txt#L13).